csrc/anemone_mempool.c

```diff
@@ -48,6 +49,8 @@ void anemone_mempool_add_block (anemone_mempool_t *pool)
 // This happens when there is not enough space in the current block for the requested bytes.
 void * anemone_mempool_alloc_block (anemone_mempool_t *pool, size_t num_bytes)
 {
+  num_bytes = round_up_allocation(num_bytes);
```

Comment:
The rounding here isn't necessary since it's already rounded up if called from `anemone_mempool_alloc`